me.njean.shootingpactris.json

```diff
@@ -0,0 +1,98 @@
+{
+    "app-id": "me.njean.shootingpactris",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "shpactris",
+    "finish-args": [
```

Comment:
Nitpick: the finish-args indentations feels weird

Reply:
Indeed. Fixed.